woodwork/column_accessor.py

```diff
@@ -30,6 +30,17 @@
 ks = import_or_none('databricks.koalas')
 
 
+def _check_series_schema(func):
+    '''Decorator for WoodworkColumnAccessor that checks schema initialization'''
+    def wrapper(self, *args, **kwargs):
+        if self._schema is None:
```

Comment:
Would it be worth testing the behavior where someone used this decorator on a method that's not on the Series Accessor? Definitely an edge case, and it's only used internally, so probably not necessary, but I just wanted to bring it up.

Reply:
Yeah I suppose people could do that in theory.  The decorator will work on any object that has a `_schema` property.  Were you thinking that we should add an `isinstance` check or something to prevent people from using it on methods from the wrong object?  I would personally say it's probably not worth doing that.  The underscore prefix I think indicate pretty clearly that it's not intended for public use.